src/lib/theme.css

```diff
@@ -1,3 +1,76 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+/* Typography */
+@font-face {
```

Comment:
These type definitions should be on a separate file that we can export on this library and then import on the Next sites (along with the font files themselves.)